Statut_HSP.tex

```diff
@@ -168,19 +174,47 @@ \section{Członkowie Stowarzyszenia}
   regularnego opłacania składek.
 \end{enumerate}
 
-\myparagraph{§ 14}
+\myparagraph{§ 12.NZ.1}
+\begin{enumerate}
+\item
+  Prawa i obowiązki każdego członka nadzwyczajnego z osobna definiuje zarząd.
+\item
+  Prawa i obowiązki członka nadzwyczajnego mogą być zmienione na jego wniosek po zatwierdzeniu przez zarząd.
+\item
+  Prawa członka nadzwyczajnego muszą zawierać się w zbiorze praw członka zwyczajnego.
+\item
+  Członek nadzwyczajny musi mieć obowiązek regularnego opłacania składek członkowskich.
+\item
+  Członek nadzwyczajny musi mieć obowiązek przestrzegania statutu i uchwał władz stowarzyszenia.
+\item 
+  Członek nadzwyczajny musi nie posiadać biernego oraz czynnego prawa wyborczego.
```

Comment:
> nie może posiadać
wiadomo, jeszcze będzie to przepisywane.

Warto sprawdzić jakie obowiązki/prawa pozostają do wyboru w części wspólnej "muszą zawierać się w zbiorze praw członka zwyczajnego" oraz powyższych domyślnych obowiązków. Chyba niedużo.

Reply:
Poprawie sformułowanie.

Ja się cykam robić za dużych eksperymentów tutaj właśnie - więc najlepiej byłoby jakby były zwrócone uwagi, na to co jest krytyczne, tj. potencjalnym blokerem.